apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialog.tsx

```diff
@@ -397,48 +397,43 @@ export const ScreeningDecisionDialog = ({
           mode="inline"
           color={triggerColor()}
           data-h2-text-align="base(left)"
+          data-h2-vertical-align="base(middle)"
         >
-          {hasBeenAssessed ? (
-            <span>
-              {initialValues?.assessmentDecision === "noDecision" ? (
-                <p>{intl.formatMessage(commonMessages.notSure)}</p>
-              ) : (
-                <>
-                  <p>
+          <>
+            {!hasBeenAssessed && poolSkill?.type === PoolSkillType.Nonessential
+              ? intl.formatMessage(poolCandidateMessages.unclaimed)
+              : intl.formatMessage(poolCandidateMessages.toAssess)}
+
+            {initialValues?.assessmentDecision === "noDecision" ? (
+              <>{intl.formatMessage(commonMessages.notSure)}</>
+            ) : (
+              <>
+                {intl.formatMessage(
+                  initialValues?.assessmentDecision
+                    ? getTableAssessmentDecision(
+                        initialValues.assessmentDecision,
+                      )
+                    : commonMessages.notFound,
+                )}
+                {initialValues?.assessmentDecision ===
+                  AssessmentDecision.Successful && !educationRequirement ? (
+                  <span
+                    data-h2-color="base(gray.darker)"
+                    data-h2-text-decoration="base(none)"
+                    data-h2-display="base(block)"
+                  >
                     {intl.formatMessage(
-                      initialValues?.assessmentDecision
-                        ? getTableAssessmentDecision(
-                            initialValues.assessmentDecision,
+                      initialValues?.assessmentDecisionLevel
+                        ? getAssessmentDecisionLevel(
+                            initialValues.assessmentDecisionLevel,
                           )
                         : commonMessages.notFound,
                     )}
-                  </p>
-                  {initialValues?.assessmentDecision ===
-                    AssessmentDecision.Successful && !educationRequirement ? (
-                    <p
-                      data-h2-color="base(gray.darker)"
-                      data-h2-text-decoration="base(none)"
-                    >
-                      {intl.formatMessage(
-                        initialValues?.assessmentDecisionLevel
-                          ? getAssessmentDecisionLevel(
-                              initialValues.assessmentDecisionLevel,
-                            )
-                          : commonMessages.notFound,
-                      )}
-                    </p>
-                  ) : null}
-                </>
-              )}
-            </span>
-          ) : (
-            <p>
-              {poolSkill?.type === PoolSkillType.Nonessential &&
-              !experienceAttachedToSkill
```

Comment:
We seem to have lost this logic

Reply:
Fixed by [9c329d2](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10016/commits/9c329d25391d81d6f166a469710972dabb130f77)

Reply:
Hmm, not sure if that is quite right. We may need to get the gang together and work through this one :sweat:

Reply:
That was a test, you passed 🥇 This is the real fix [f4dca7a](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10016/commits/f4dca7a27ae8273fb650ded004b71acb7b9c9ff4)